RELEASE.md

```diff
@@ -1,14 +1,59 @@
+1. Describe the notable changes to the stack as release notes
+1. Tick _This is a pre-release_ checkbox
+1. Click _Publish release_ button
 
-[Travis-CI](https://travis-ci.org) is configured to build and test the stacks and indexes on GitHub pull requests. When a GitHub release is created manually, the built and tested stacks and indexes are published on DockerHub and GitHub releases. 
+### Monitor the Travis build
+1. Watch the [Travis build](https://travis-ci.com/appsody/stacks) for the release and ensure it passes
+1. Check the release artefacts to ensure these all exist:
+    - archives for all the templates for the released stack on [GitHub releases](https://github.com/appsody/stacks/releases)
+    - latest indexes for each of the repos (`incubator`, `experimental`, `stable`) on [GitHub releases](https://github.com/appsody/stacks/releases)
+    - updated docker hub images with appropriate tags on [dockerhub](https://hub.docker.com/u/appsody)
 
-## Build
+### Update the pre-release
+1. Navigate to https://github.com/appsody/stacks/releases
+1. Click on the title of the pre-release you have just created
+1. Click _Edit release_ button
+1. Untick _This is a pre-release_ checkbox
+1. Click _Edit release_ button
 
-The `build.sh` script is a wrapper created for convenience and covers the process of identifying which stacks need to be built, builds stack images and templates, generates all repo indexes and then tests the stacks and indexes locally. 
+### Update v1 Index
+1. For now we are manually updating the [old appsody stacks index](https://github.com/appsody/stacks/blob/master/index.yaml)
+1. Create a pull request to update this file and update the index manually for the updated stack. Ususally, we only need to update the `version` and `url` fields for the updated stack
+1. Submit the pull request
+1. Once the pull request is approved, merge the changes
+
+
+# Releasing Dependent Stacks
+There is no fixed release schedule for Appsody stacks. Stacks should be released as we merge pull requests in this repo.
+
+If a stack is consumed by other stacks, then we must also update the dependent stacks and release new versions of those. For example, a change to `nodejs` stack will need a new release of `nodejs`, followed by a new release of `nodejs-express`.
+
+# Appsody Stacks Release Process - Technical Overview
+
+This document outlines the process for building, testing and releasing Appsody stacks.
+
+`appsody/stacks` contains multiple stacks repos such as `experimental`, `incubator` and `stable`. Each repo includes several stacks. The build process allows you to build and release each stack and the indexes for each stack repo.
+
+[Travis-CI](https://travis-ci.org) is configured to build and test the stacks and indexes on GitHub pull requests. When a GitHub release is created manually, the built and tested stacks and indexes are published on DockerHub and GitHub releases.
```

Comment:
could we change this link to https://travis-ci.com, 
I encountered a couple of problems with the .org

Reply:
done

Reply:
done